roles/keystone/tasks/main.yml

```diff
@@ -31,9 +31,49 @@
     - etype: other
       permission: r
 
-- name: install keystone service
-  upstart_service: name=keystone user=keystone cmd=/usr/local/bin/keystone-all
-                   config_dirs=/etc/keystone
+- name: install keystone uwsgi service
+  template: src=roles/keystone/templates/etc/init/keystone.conf
```

Comment:
Ah, right. I recall that was a need for something else too. But the swift service does it by template, so might as well continue that here for now.